zerver/lib/widget.py

```diff
@@ -29,18 +37,43 @@ def get_extra_data_from_widget_type(content: str,
         options = []
         if lines and lines[0]:
             question = lines.pop(0).strip()
-        for line in lines:
-            # If someone is using the list syntax, we remove it
-            # before adding an option.
-            option = re.sub(r'(\s*[-*]?\s*)', '', line.strip(), 1)
-            if len(option) > 0:
-                options.append(option)
+
+        def append_option(option: str) -> None:
+            options.append(option)
+
+        get_widget_lines_of_text(lines, append_option)
         extra_data = {
             'question': question,
             'options': options,
         }
         return extra_data
-    return None
+    if widget_type == 'todo':
+        todos: Any = []
```

Comment:
Not sure how I can type this better than "Any"

Reply:
List[str] doesn’t work?